pkgs/build-support/fetchurl/default.nix

```diff
@@ -57,6 +57,11 @@ in
   # first element of `urls').
   name ? ""
 
+  # for versioned downloads optionally take pname + version.
+  # if name is set that is preferred
```

Comment:
What «that» is preferred? Doesn't later code prefer `pname` and `version` by putting them into the first branch to check?

(I agree with what the code does, and I don't have problems with not having this line of the comment at all, I just fail to understand it as written)